components/org.wso2.carbon.identity.api.user.session/org.wso2.carbon.identity.api.user.session.v1/src/main/java/org/wso2/carbon/identity/rest/api/user/session/v1/impl/UserIdApiServiceImpl.java

```diff
@@ -61,11 +65,27 @@ public Response terminateSessionBySessionId(String userId, String sessionId) {
     }
 
     @Override
-    public Response terminateSessionsByUserId(String userId) {
+    public Response terminateSessionsByUserId(String userId) throws UserStoreException {
```

Comment:
catch the UserStoreException instead of throwing and generate a proper Response object instead.

Reply:
Fixed